test/unit/question_base_test.rb

```diff
@@ -8,7 +8,7 @@ class QuestionBaseTest < ActiveSupport::TestCase
   context '#next_node' do
     should 'raise exception if called with block but no permitted next nodes' do
       e = assert_raises(ArgumentError) do
-        @question.next_node do
+        @question.next_node(permitted: []) do
           :done
```

Comment:
No, although I see why you ask. This is still testing the "old" mechanism which doesn't require the wrapping of the result with `question` or `outcome`. I hope to remove this functionality soon.